lib/game/player_type/hard_mode_computer.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Game.PlayerType.HardComputerPlayer do
```

Comment:
You might want to address these in a separate PR, but the filepath/name for this module is `lib/game/player_type/hard_mode_computer.ex` while the test is`test/game/player_type_tests/hard_computer_player_test.exs`.

I would remove all of the `_tests` suffixes in your test directories and only include that in your test files themselves by convention. Also there is a naming mismatch between the production file and the test file:

For `hard_mode_computer.ex` I would expect the test filename to be `hard_mode_computer_test.exs` and not
`hard_computer_player_test.exs`.

Reply:
On it! I can't believe I didn't catch that mismatch